docs/doc.md

```diff
@@ -1,7 +1,6 @@
-/*
-Package search implements a database text search engine appropriate to IMQS data.
+# Package search implements a database text search engine appropriate to IMQS data.
```

Comment:
This package was decent with godocs, so rather just leave it the way it was.
See https://godoc.org/github.com/IMQS/search-builder/search